lib/IRGen/GenDecl.cpp

```diff
@@ -29,7 +29,7 @@
 #include "swift/ClangImporter/ClangModule.h"
 #include "swift/Demangling/ManglingMacros.h"
 #include "swift/IRGen/Linking.h"
-#include "swift/Runtime/HeapObject.h"
+#include "../../stdlib/public/SwiftShims/RefCount.h"
```

Comment:
We could replace this with "swift/ABI/HeapObject.h" that @jckarter recently introduced (this file also transitively includes SwiftShims ... but at least not more)